internal/kafka/command_cluster_describe.go

```
@@ -148,14 +150,32 @@ func convertClusterToDescribeStruct(cluster *cmkv2.CmkV2Cluster, ctx *config.Con
 		EncryptionKeyId:    getCmkEncryptionKey(cluster),
 		RestEndpoint:       cluster.Spec.GetHttpEndpoint(),
 	}
+
+	// Only set limits field if usage limits are available
+	// For the list command, no limits are displayed in the command output so usageLimits is nil
+	if usageLimits != nil && out.Type != "UNKNOWN" {
+		maxEcku := getCmkMaxEcku(cluster)
```

Nit: let's add a condition to check if `getCmkMaxEcku > 0` and return an error message if not

---

why do we need to return an error? Not all MT clusters have max ecku set and that is expected behavior. I already check that when using the value here https://github.com/confluentinc/cli/pull/3217/files#diff-8de7b6576e3519cffacc178f5a4e2e78b74456d23b4f2edff203e1ca5d0f7787R87

---

That make sense, thanks